src/models/listing-type.js

```diff
@@ -0,0 +1,36 @@
+const { Sequelize } = require('sequelize');
+const BaseModel = require('models/base-model');
+const getSharedColumns = require('models/shared-columns');
+
+class ListingType extends BaseModel {
+  static associate({ Listing }) {
+    this.hasOne(Listing, { as: 'listing', foreignKey: 'listingTypeId' });
```

Comment:
lookl ike this is a `1:n` relation, i think we should use `hasMany` here
```suggestion
    this.hasMany(Listing, { as: 'listings', foreignKey: 'propertyTypeId' });
```